### PR TITLE
Add port_scans to redshift pull

### PR DESCRIPTION
**Pull request by DJensen94**
Pull port scans from redshift db
## 🗣 Description ##
Pull port scan data from redshift for the last 2 days and save to the data lake
## 💭 Motivation and context ##
This data will be used to recreate parts of the VS report in Crossfeed

## 🧪 Testing ##
Tested locally off of locally pulled data

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [X] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [X] All relevant type-of-change labels have been added.
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [X] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [X] All relevant repo and/or project documentation has been updated

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: backend/src/tasks/vs_sync.ts [hunk 1]
```diff
@@ -566,4 +568,74 @@ export const handler = async (commandOptions: CommandOptions) => {
...
+          organization: { id: org_id_dict[port_scan.owner] },
+          latest: port_scan.latest,
+          port: port_scan.port,
+          protocol: port_scan.protocol,
+          reason: port_scan.reason,
+          service: port_scan.service,
+          source: port_scan.source,
+          state: port_scan.state,
+          timeScanned: port_scan.time
+          // @ManyToMany((type) => Snapshot, (snapshot) => snapshot.portScans, {
```
Comment (cduhn17) [inline backend/src/tasks/vs_sync.ts]: If your keeping this for future use, create ticket and note in comment above

**Review by cduhn17 — COMMENTED** [review]
See comment regarding commented out code

File: backend/src/tasks/vs_sync.ts (hunk 1 above)
Comment (DJensen94) [reply backend/src/tasks/vs_sync.ts]: addressed in this commit: [c0d3b56](https://github.com/cisagov/XFD/pull/258/commits/c0d3b5674065740a0a6f09b2258d89ea1a7f2ba7)

**Review by cduhn17 — APPROVED** [review]
LGTM

**Review by aloftus23 — APPROVED** [review]
Looks good

**Review by schmelz21 — APPROVED** [review]

**Pull request merged**